applications/RomApplication/python_scripts/calculate_nodal_area_process.py

```diff
@@ -0,0 +1,39 @@
+import KratosMultiphysics
```

Comment:
Rather than implementing this, I think it makes sense to add a `recompute_nodal_area` flag to the base `CalculateNodalAreaProcess`. This flag is set to `false` by default, so there is no behaviour change.

Reply:
This has not been changed

Reply:
Then why are you adding this rather than using the one in the core?

Reply:
I might be wrong, but there is no python script in the core for using the calculate_nodal_area_process from the ProjectParameters.json. The only one was in the SwimmingDEMApplication. 

We are using the core implementation, through this python script to be able to put the process in the ProjectParameters.json. Now this script is in these two applications; but it is missing in the core...

Reply:
My point is to directly use the one in the core as it is exported to Python. I don't think it makes too much sense to expose this kind of processes to the Python.

In any case this problem should disappear as soon as we register the processes.